inst/scripts/make_tritryp_pkgs.R

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env Rscript
+## Collect eupathdb data from their various webservices and make packages from it.
```

Comment:
What's the purpose for having a separate `make_tritryp_pkgs.R` script? (Compared with everything else, presumably handled by `make_eupath_pkgs.R`?)

Reply:
Nothing really, but I wanted to explicitly test the tritrypdb stuff since that is the  service I use.  The thing to do is perhaps to have make_eupath_pkgs.R use getopt and have a webservice argument which defaults somewhere useful.